management/prompt_test.go

```diff
@@ -2,8 +2,12 @@ package management
 
 import (
 	"gopkg.in/auth0.v5"
+	"encoding/json"
 	"gopkg.in/auth0.v5/internal/testing/expect"
+
 	"testing"
+
+	"gopkg.in/auth0.v5/internal/testing/expect"
```

Comment:
this line seems duplicated with line 6

Reply:
Thanks. I have removed this duplicate line.